resources/js/processes-catalogue/components/menuCatologue.vue

```diff
@@ -82,24 +88,50 @@
         </b-list-group-item>
       </b-list-group>
     </b-collapse>
+
+    <select-template-modal
+      :type="$t('Process')"
+      :countCategories="countCategories"
+      ref="addProcessModal"
+      hide-add-btn="true"
+    >
+    </select-template-modal>
   </div>
 </template>
 
 <script>
 import SearchCategories from "./utils/SearchCategories.vue";
+import SelectTemplateModal from "../../components/templates/SelectTemplateModal.vue";
 
 export default {
   components: {
     SearchCategories,
+    SelectTemplateModal,
   },
-  props: ["data", "select", "title", "preicon", "filterCategories", "fromProcessList"],
+  props: [
+    "data",
+    "select",
+    "title",
+    "preicon",
+    "filterCategories",
+    "fromProcessList",
+  ],
   data() {
     return {
+      showTemplateModal: true,
+      assetName: null,
+      assetId: null,
+      modalProcess: true,
+      countCategories: 0,
       showCatalogue: false,
       showGuidedTemplates: false,
       selectedProcessItem: null,
       selectedTemplateItem: null,
       templateOptions: [
+        {
+          label: this.$t("All Templates"),
```

Comment:
Please add this new translation into the en.json

Reply:
solved